generate.js

```diff
@@ -43,6 +46,8 @@ const template = `
   </head>
   <body>
     <h1>GCI Current Leaders</h1>
+    <small id="last-generated"></small>
```

Comment:
Please put the UTC date as well, like in http://community.coala.io/
Feel free to take a look at it's source code in coala/community.

Reply:
@Indmind ^ Please do that first